src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25671.cs

```diff
@@ -42,13 +42,13 @@ public async Task LayoutPassesShouldNotIncrease()
 			var arrangePasses = int.Parse(match.Groups[2].Value);
 
 #if IOS
-            var maxMeasurePasses = 225;
-            var maxArrangePasses = 247;
+			var maxMeasurePasses = 203;
```

Comment:
why do the values change here ?

Reply:
Layout passes decreased, which means we can lower those values and ensure no performance regression in the future!